docs/docs/user-guide/relabeling.mdx

```diff
@@ -12,13 +12,19 @@ This empowers annotators to address any missing elements and enhance the overall
 
 ## Steps to send data to the labeling stage
 
-0. **[Optional]** Use the [filter feature](../user-guide/filtering) from the toolbox in the _Data Quality_ or _Label Quality_ _Explorer_ section to choose the desired data. 
+0. **[Optional]** Use the [filter feature](../user-guide/filtering) from the toolbox in the _Data Quality_ or _Label Quality_ _Explorer_ section to choose the desired data.
+   ![data-quality-explorer-filter-by-tag](../images/user-guide/data-quality-explorer-filter-by-tag.png)
 1. Access the Action tab from the toolbox in the _Data Quality_ or _Label Quality_ _Explorer_ section.
+   ![toolbox-action-tab](../images/user-guide/toolbox-action-tab.png)
 2. Click the _🖋 Re-label_ button.
+   ![toolbox-action-tab-relabel-button](../images/user-guide/toolbox-action-tab-relabel-button.png)
 3. Verify the number of tasks ready for submission to the labeling stage is correct and press the _Confirm_ button.  
+   ![toolbox-action-tab-relabel-button-confirmed](../images/user-guide/toolbox-action-tab-relabel-button-confirmed.png)
 4. **[Optional]** Navigate to the project in Encord Annotate and explore the task modifications in the _Summary_ tab.
-
-![action-tab-in-toolbox](../images/user-guide/action-tab-in-toolbox.png)
+   <div style={{display: "flex", justifyContent: "space-between"}}>
+     <img src="https://raw.githubusercontent.com/encord-team/encord-active/docs/add-screenshots-in-relabel-feature/docs/docs/images/user-guide/workflow-project-summary.png" width="48%" style={{marginRight: "4%"}} alt="workflow-project-summary.png"/>
+     <img src="https://raw.githubusercontent.com/encord-team/encord-active/docs/add-screenshots-in-relabel-feature/docs/docs/images/user-guide/workflow-project-summary-after-relabeling.png" width="48%" alt="workflow-project-summary-after-relabeling.png"/>
+   </div>
```

Comment:
If you move the images to `docs/docs/static/img/user-guide`, I believe that you could do something like this to display them side by side with captions:

```suggestion
<div style={{ display: "flex", justifyContent: "space-around", gap: "2em" }}>
  <div style={{ width: "45%", textAlign: "center" }}>
    <img
      src="../img/user-guide/workflow-project-summary.png"
      width="100%"
      alt="workflow-project-summary.png"
    />
    <p>Before caption</p>
  </div>
  <div style={{ width: "45%", textAlign: "center" }}>
    <img
      src="../img/user-guide/workflow-project-summary-after-relabeling.png"
      width="100%"
      alt="workflow-project-summary-after-relabeling.png"
    />
    <p>After caption</p>
  </div>
</div>
```

Reply:
it works great! thanks!!